src/components/record/related-table-actions.tsx

```diff
@@ -553,39 +520,17 @@ const RelatedTableActions = ({
           dispatchError({ error: err, isDismissible: true });
         };
 
-        if (!CONFIRM_DELETE) {
```

Comment:
For unlink, we are not trying to remove the delete confirmation modal. We only want to remove the modal with title "Batch Unlink Summary" that shows after clicking "Unlink". The confirm delete modal is a conditional popup that is configured in chaise-config.

The other change is the alert after success which you did remove properly